Hardcore.lua

```diff
@@ -73,7 +73,7 @@ Hardcore_Character = {
 }
 
 --[[ Local variables ]]--
-local debug = false
+local debug = true
```

Comment:
should probably revert debug back to false before merging

Reply:
Reverted to false